src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -195,6 +195,8 @@
    :state                                          s/Str
    :score                                          (s/maybe s/Int)
    :new-application-modifications                  s/Int
+   :person                                         {:preferred-name s/Str
+                                                    :last-name      s/Str}
    (s/optional-key :form)                          s/Int
    (s/optional-key :preferred-name)                (s/maybe s/Str)
```

Comment:
Näitä avaimia ei varmaankaa tarvita?

Reply:
Tarvitaan siinä hakemuslistauskessa, ja nääkin halutaan ONR:stä

Reply:
Tarkotin nuita `:preferred-name` ja `:last-name` jotka on jo tuolla uudessa `:person` osassa.